client/src/document/on-github.tsx

```diff
@@ -15,13 +15,13 @@ export function OnGitHubLink({ doc }: { doc: Doc }) {
           <NewIssueOnGitHubLink doc={doc} />
         </li>
         <li>
-          Want to fix the problem yourself? See{" "}
+          Want to fix the problem yourself? See the{" "}
           <a
-            href="https://github.com/mdn/content/blob/main/README.md"
+            href="https://github.com/mdn/content/blob/main/CONTRIBUTING.md"
             target="_blank"
             rel="noopener noreferrer"
           >
-            our Contribution guide
+            MDN Web Docs contribution guide
```

Comment:
My only concern is that this more verbose and significantly longer than before:

```
See our Contribution guide
See the MDN Web Docs contribution guide
```

Do we really need to mention that it's the "MDN Web Docs" contribution guide?

Reply:
PS: I also feel like "**our** contribution guide" sounds more friendly.

Reply:
How about "Learn [how to contribute](https://github.com/mdn/content/blob/main/CONTRIBUTING.md)!" instead, avoiding the technical term?

Reply:
done in 33cb33b93e3ae3cbd18ef02c51c2d4897d600a10